packages/util-eslint-runner/package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "@springernature/util-eslint-runner",
   "description": "Run ESLint only if Javascript files are found",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/springernature/frontend-toolkit-utilities.git"
```

Comment:
what is the git+ bit at the beginning of the url?

Reply:
This specifies the protocol, this is how `npm init` generates a repository link
https://docs.npmjs.com/cli/install#description